ignite/distributed/utils.py

```diff
@@ -480,12 +480,15 @@ def initialize(backend: str, **kwargs: Any) -> None:
             - | "horovod" : comm(=None), more info: `hvd_init`_.
 
     Examples:
-        Launch single node multi-GPU training with ``torch.distributed.launch`` utility.
+        Launch single node multi-GPU training with ``torchrun`` or ``torch.distributed.launch`` utility.
```

Comment:
```suggestion
        Launch single node multi-GPU training with ``torchrun`` or ``torch.distributed.run`` utility.
```
As `torch.distributed.launch` is deprecated, let's just mention `run`: torchrun or torch.distributed.run

Reply:
should we do this in every occurrence of `torch.distributed.launch` as well?

Reply:
I commented where I think it would be better to remove `torch.distributed.launch` and where we can just rename `launch` by `run`.